app/controllers/users_controller.rb

```diff
@@ -1,5 +1,30 @@
 class UsersController < ApplicationController
-  attr_accessor :name, :email
+  before_action :load_user, only: :show
 
-  def new; end
+  def show; end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = I18n.t("static_pages.home.title")
```

Comment:
```suggestion
      flash[:success] = t "static_pages.home.title"
```
trong controller không cần viết rõ I18n nhé